specs-go/config.go

```diff
@@ -27,6 +27,8 @@ type Spec struct {
 	Windows *Windows `json:"windows,omitempty" platform:"windows"`
 	// VM specifies configuration for virtual-machine-based containers.
 	VM *VM `json:"vm,omitempty" platform:"vm"`
+	// Zos is platform-specific configuration for z/OS based containers.
+	Zos *Zos `json:"zos,omitempty" platform:"zos"`
```

Comment:
In Go, [initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) generally remain capitalized.  Seems like the field (and type) should be `ZOS` instead of `Zos`.

```suggestion
	// ZOS is platform-specific configuration for z/OS based containers.
	ZOS *ZOS `json:"zos,omitempty" platform:"zos"`
```